play-services-api/src/main/aidl/com/google/android/gms/potokens/internal/IPoTokensService.aidl

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: 2023 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.google.android.gms.potokens.internal;
+
+import com.google.android.gms.common.api.internal.IStatusCallback;
+import com.google.android.gms.potokens.internal.ITokenCallbacks;
+
+interface IPoTokensService {
+    void responseStatus(IStatusCallback call, int code) = 1;
+    // The direction of 'bArr' is not specified. array can be an in, out, or inout parameter.
```

Comment:
No need to state that. For complex types like arrays or objects, direction needs to be specified. For simple native types like ints and Strings, they are redundant.

Reply:
okay, changed